src/Microsoft.ML.Data/DataView/DataViewConstructionUtils.cs

```diff
@@ -46,6 +46,57 @@ internal static class DataViewConstructionUtils
             return new StreamingDataView<TRow>(env, data, internalSchemaDefn);
         }
 
+        public static StreamingDataView<TRow> CreateFromEnumerable<TRow>(IHostEnvironment env, IEnumerable<TRow> data,
```

Comment:
>CreateFromEnumerable [](start = 46, length = 20)

So, I don['t object exactly, it seems positive, but I'm not sure how this relates to the original issue?

Reply:
I added these so that I could add tests show what you can do with the schema once you load it with the new API.

---
In reply to: [266546399](https://github.com/dotnet/machinelearning/pull/2858#discussion_r266546399) [](ancestors = 266546399)